pertpy/tools/_distances/_distances.py

```diff
@@ -80,6 +114,9 @@ class Distance:
         Average of the classification probability of the perturbation for a binary classifier.
     - "classifier_cp": classifier class projection
         Average of the class
+    - "mean_var_distn": Distance between mean-var distibutions of gene expression.
+    - "mahalanobis": Mahalanobis distance between the cells of two groups.
```

Comment:
This might requires one more sentence of explanation or link to what this distance actually is